src/components/common/helpers/compareVersion.ts

```diff
@@ -10,17 +9,27 @@ export function getVersionArr(version: string): number[] {
     return [Number(versionStr[0]), Number(versionStr[1])];
 }
 
-export function isVersionLessThanOrEqualToTarget(version: string, target: number[]): boolean {
+export function isVersionLessThanOrEqualToTarget(version: number[], target: number[]): boolean {
```

Comment:
simply logic is this function

Reply:
This function doesn't do any type conversion of version anymore. contains only logic for comparison of versions (represented as array of numbers).